yt_dlp/extractor/vidio.py

```diff
@@ -56,9 +60,28 @@ def _real_extract(self, url):
             })
         video = data['videos'][0]
         title = video['title'].strip()
+        hls_url = data['clips'][0]['hls_url']
```

Comment:
is this key guaranteed to be present even for premium videos? If not, use `try_get`

Reply:
now irrelevant because premium is checked before `hls_url` is referenced